plc4j/api/src/main/java/org/apache/plc4x/java/api/value/PlcValue.java

```diff
@@ -149,4 +149,10 @@ public interface PlcValue {
 
     Map<String, ? extends PlcValue> getStruct();
 
+    Map<String, PlcValue> getMetaData();
+
+    boolean hasMetaData(String key);
```

Comment:
not sure if those are worth it if you expose a map anyway. This just bloats the API